client/src/stores/UserStore.js

```diff
@@ -13,6 +14,12 @@ var UserStore = Reflux.createStore({
   init: function() {
     this.user = null
   },
+  onUsernameSelected: function(username) {
+    logger.debug('username selected')
+    this.user = {
+      name: username
+    }
+  },
```

Comment:
We can't put the username here as such. The idea of this store is that it wraps around `orbit.user` one-way, ie. when the user changes in Orbit, it changes here, but not changing it here manually. 

Perhaps we should do it something like this:

In App.js:
```javascript
onUsernameSelected: function(username) {
  const stop = AppActions.hasInitialized.listen(() => {
    stop()
    NetworkActions.connect(null, username)
  })
  const stopListening = IpfsDaemonActions.daemonStarted.listen((ipfs) => {
    stopListening()
    IpfsDaemonActions.daemonStarted(ipfs)
    const orbit = new Orbit(ipfs)
    // rest of Orbit init code, listen for 'connect' event, etc
    AppActions.initialize(orbit);
  })
  IpfsDaemonActions.start();
}
```

That way we start the daemon init process when the user presses 'connect' on the login screen, can init Orbit once daemon is there and don't need to save the login username anywhere in the app.

Does that make sense?

I would also change the name of the action `onUsernameSelected` to `onLogin`.